docs/docs/0.3.2/policies/README.md

```diff
@@ -425,6 +429,60 @@ In particular, `prometheus.io/*` annotations are part of configuration used by `
 If you're using a different way to install `Prometheus` on `kubernetes`, those annotations might not have the desired effect.
 :::
 
+Although it's easy and works without any extra configuration, those annotations supports only one address to scrape.
+If you also have an application next to Kuma DP which exposes metrics through `prometheus.io/*` annotations,
+you have to use the other way to expose metrics for both the application and Kuma DP.
+
+##### Kuma Prometheus SD
+
+You can deploy `kuma-prometheus-sd` container next to you Prometheus instance just like in Universal setup.
+
+First, add a volume to your Prometheus deployment to which `kuma-prometheus-sd` will write a file with the list of the dataplanes and from which `Prometheus` will read the list.
+
+```yaml
+volumes:
+- name: kuma-prometheus-sd-volume
+  emptyDir: {}
+```
+
+Then add a new container with `kuma-prometheus-sd`. It will connect to the control plane at given address and produce file to `/var/run/kuma.io/prometheus-sd/kuma.file_sd.json` in created volume.
+
+```yaml
+  containers:
+- name: kuma-prometheus-sd
+  image: kong-docker-kuma-docker.bintray.io/kuma-prometheus-sd:0.3.2
+  imagePullPolicy: Always
+  args:
+    - run
+    - --name=kuma-prometheus-sd
+    - --cp-address=http://kuma-control-plane.kuma-system:5681
+    - --output-file=/var/run/kuma.io/prometheus-sd/kuma.file_sd.json
+  volumeMounts:
+    - mountPath: "/var/run/kuma.io/prometheus-sd"
+      name: kuma-prometheus-sd-volume
+```
+
+Next step is to mount the volume to the Prometheus container
+```yaml
+volumeMounts:
+- mountPath: "/var/run/kuma.io/prometheus-sd"
+  name: kuma-prometheus-sd-volume
+```
+
+Finally, modify your Prometheus config to use generated file
+```yaml
+- job_name: 'kuma-dataplanes'
+  scrape_interval: "5s"
+  file_sd_configs:
+  - refresh_interval: 1s
```

Comment:
Let's remove  `refresh_interval: 1s` from the example.

It was pointed out by Prometheus maintainers that the primary trigger for refresh events are changes to the file rather than fixed interval. `refresh_interval` is only a safety net, default value `5 min` is more appropriate than `1 sec`